### DISCUSSION TITLE
Custom blockchains

### DISCUSSION THREAD

**Discussion by yovanoc** [bitcoindevkit/bdk#1629] [Ideas]
I don't know if there is already a way, I didn't found it, but to use bdk for 'forked' bitcoin chains like evrmore, its juste a legacy bitcoin fork, but with custom script params. I didn't find how to specify the custom things.
Thanks a lot

**Comment by storopoli** [comment]
Barking in the wrong tree buddy...

https://github.com/bitcoindevkit/bdk/blob/2cf46a2a9e513d3f2612b7c363446fb1acf958ae/README.md?plain=1#L31-L32

Try to change these core deps, which in my opinion highly unlikely.
Don't know about evermore and don't know if I should pay attention to it.

**Reply by yovanoc** [reply]
I didn't wanted to being rude. Just asked if it was something possible because evrmore is just bitcoin with different parameters, and wanted to know if someone has done something similar with another bitcoin fork

**Reply by storopoli** [reply]
So just fork or patch `rust-bitcoin` and `miniscript`. Probably 95% of what BDK needs would be there.
I don't have much information about bitcoin forks.

**Reply by jaoleal** [reply]
@yovanoc Hopefully someone already did the fork and maintain a lib for a specific "bitcoin fork", if not, its your chance to do so... The problem of having this kind of feature is to maintain it and, as you can see, people here really likes bitcoin and dont like this technology being used on other "coins" because of the scam nature they have nowadays... People can get a little upset about it.
